### DISCUSSION TITLE
Dashboard 3.12 Release

### DISCUSSION THREAD

**Discussion by karolkielecki** [mirumee/saleor-dashboard#3302] [General]
Hi Community!

Welcome to the Dashboard 3.12 release! We've introduced the first batch of changes in the dashboard redesign. 
We're also working on a new version of our UI library - MacawUI. 

Read more in the changelog: https://github.com/saleor/saleor-dashboard/releases/tag/3.12.0

**Comment by ghost** [comment]
Looks great, is there a Figma link to the new dashboard that I can check out?
